assignments/arrays.js

```diff
@@ -63,34 +63,87 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+let year;
+let make;
+let model;
+
+for (let i = 0; i < inventory.length; i++) {
+  let currentCar = inventory[i];
+  if (currentCar[`id`] === 33) {
+    year = currentCar.car_year;
+    make = currentCar.car_make;
+    model = currentCar.car_model;
+  }
+}
+
+console.log(`Car 33 is a ${year} ${make} ${model}`);
```

Comment:
I appreciate the resourcefulness and enthusiasm, well done! Josh was just looking for a console log like 

`console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );`